src/interface/src/styleguide/debounce-input/debounce-input.component.scss

```diff
@@ -55,20 +55,35 @@
     color: $color-dark-red;
     height: 24px;
   }
+
+  .save-spinner {
+    right: 10px;
+  }
 }
 
 ::ng-deep .name-input-component input {
   width: 100%;
 }
 
+::ng-deep .name-input.small input:disabled {
```

Comment:
you could simplify this with 
::ng-deep.name-input.small {
  input:disabled {...}
  .input-wrapper {...}
  input-wrapper:focus-within 
}